### ISSUE TITLE
Update timing tables for land-only compsets

### ISSUE THREAD

**Issue by samsrabin** [ESCOMP/CTSM#2722]
https://cseg.cgd.ucar.edu/timing/timings/

**Comment by samsrabin** [comment]
May depend on #2745, if we want to include certain FATES compsets.

**Comment by samsrabin** [comment]
#1076 is a child of this issue.

**Comment by slevis-lmwg** [comment]
Timing-table updates help with computer time application proposals and other proposal writing. I will treat this as low priority until I'm told that we need it soon or now.

UPDATE: Also it will make more sense to proceed with this task once we have the compsets that we're releasing with the model.
